src/services/catalog.service.ts

```diff
@@ -158,6 +160,7 @@ export class ImageItem implements IServiceItem {
     this.longDescription = this.getLongDescription();
     this.catsBySubCats = this.getCategoriesBySubCategories();
     this.isBuilder = _.includes(this.tags, 'builder');
```

Comment:
So, I'm using 
`this.tags = this.catalogSrv.$filter('imageStreamTagTags')(this.resource);`
Since I'm not specifing a tagName, it is using 'latest' as tagName.  Are you saying I need to verify that 'latest' exists in status.tags?  ...and if it doesn't, do not use the image?